Persephone/Views/CurrentArtView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CurrentArtView.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/4/27.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+import ReSwift
+
+class CurrentArtView: NSImageView {
+  required init?(coder: NSCoder) {
+    super.init(coder: coder)
+
+    AppDelegate.store.subscribe(self) {
```

Comment:
Is there a more fine-grained place for views to observe app state? For example, if the app is hidden, it might make sense to throttle or pause view updates.